src/renderer/components/+nodes/node-charts.tsx

```diff
@@ -26,9 +26,11 @@ export const NodeCharts = observer(() => {
   const [
     memoryUsage,
     memoryRequests,
+    _memoryLimits, // DO NOT REMOVE!!!
```

Comment:
I think warning can be removed by setting `varsIgnorePattern` in `no-unused-vars` configuration to something like "^_"